uavcan/equipment/ahrs/1002.RawIMU.uavcan

```diff
@@ -0,0 +1,21 @@
+#
+# Raw IMU data with timestamps
+#
+
+#
+# rad/sec
+#
+uavcan.Timestamp gyro_timestamp
+float16[3] gyro_point
+float16[3] gyro_integral
+float16 gyro_integral_dt
+float16[<=9] gyro_covariance
+
+#
+# m/sec^2
+#
+uavcan.Timestamp accel_timestamp
+float16[3] accel_point
+float16[3] accel_integral
+float16 accel_integral_dt
+float16[<=9] accel_covariance
```

Comment:
Missing trailing newline. Note that no alignment is needed here, because tail array optimization will elide the size field.